### ISSUE TITLE
Support for HELM charts

### ISSUE THREAD

**Issue by pja-kit** [bitnami/kubecfg#267]
KSonnet had an std.native function:
`renderHelmChart`
A similar thing can be achieved by using a workaround, by doing a 
`helm template ...`
And importing the resulting YAML by using
`kubecfg.parseYaml`

However by doing this the configuration needs to be in both YAML for the helm template, and jsonnet for the kubecfg parts. Therefor it would be a really useful if kubecfg had something similar to renderHelmChart in KSonnet.

**Comment by mkmik** [comment]
Yeah this was something I'd wanted to attack for a long time now. I'm happy to see there is public interest too.
